src/EVTUI/UI/Widgets/Rendering/GFDRenderingPanelViewModel.cs

```diff
@@ -56,9 +60,40 @@ public void InitTestData()
         this.sceneManager.sceneModels.Last().StartAnimTimer();
     }
 
+    public void LoadQueuedItems()
+    {
+        string vsPath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "GFDStudio", "app_data", "shaders", "default.glsl.vs");
```

Comment:
I guess this might have to change at some point? I'm not sure we plan to distribute GFDStudio alongside the compiled binary?

Reply:
Yeah, I'm not sure what our plans are re: distribution of external dependencies. I'd prefer minimal extra setup for users, but it also depends on permissions, of course.

The whole `LoadQueuedItems` method is a very hacky solution already that should definitely be integrated more thoughtfully, if not entirely replaced, from the shader paths to the general model management. So... yes, you guess correctly, I think.